Package.swift

```diff
@@ -4,154 +4,158 @@ import PackageDescription
-                .target(name: "Apodini")
-            ]
-        ),
-        .testTarget(
-            name: "ApodiniTests",
-            dependencies: [
-                .product(name: "XCTVapor", package: "vapor"),
-                .target(name: "XCTApodini"),
-                .target(name: "ApodiniDatabase")
-            ],
-            exclude: [
-                "ConfigurationTests/Certificates/cert.pem",
-                "ConfigurationTests/Certificates/key.pem"
-            ]
-        ),
-        .target(
-            name: "TestWebService",
-            dependencies: [
-                .target(name: "Apodini"),
-                .target(name: "ApodiniDatabase")
-            ]
-        ),
-        // ProtobufferCoding
-        .target(
-            name: "ProtobufferCoding",
-            dependencies: [
-                .product(name: "Runtime", package: "Runtime")
-            ],
-            exclude:["README.md"]
-        ),
-        .testTarget(
-            name: "ProtobufferCodingTests",
-            dependencies: [
-                .target(name: "ProtobufferCoding")
-            ]
-        ),
-        // WebSocket Infrastructure
-        .target(
-            name: "WebSocketInfrastructure",
-            dependencies: [
-                .product(name: "OpenCombine", package: "OpenCombine"),
-                .product(name: "OpenCombineFoundation", package: "OpenCombine"),
-                .product(name: "Vapor", package: "vapor"),
-                .product(name: "Fluent", package: "fluent"),
-                .product(name: "NIOWebSocket", package: "swift-nio"),
-                .product(name: "AssociatedTypeRequirementsKit", package: "AssociatedTypeRequirementsKit"),
-                .product(name: "Runtime", package: "Runtime")
-            ]
-        ),
-        // Jobs
-        .target(
-            name: "Jobs",
-            dependencies: [
-                .target(name: "Apodini"),
-                .product(name: "SwifCron", package: "SwifCron")
-            ]
-        ),
-        .testTarget(
-            name: "JobsTests",
-            dependencies: [
-                .target(name: "Jobs"),
-                .target(name: "XCTApodini")
-            ]
-        ),
-        // Notifications
-        .target(
-            name: "Notifications",
-            dependencies: [
-                .target(name: "Apodini")
-            ]
-        ),
-        .testTarget(
-            name: "NotificationsTests",
-            dependencies: [
-                .product(name: "XCTVapor", package: "vapor"),
-                .target(name: "Notifications"),
-                .target(name: "XCTApodini")
-            ],
-            exclude: [
-                "Helper/mock_fcm.json",
-                "Helper/mock_invalid_fcm.json",
-                "Helper/mock.p8",
-                "Helper/mock.pem"
-            ]
-        )
-    ]
+        name: "Apodini",
```

Comment:
What's the change here? indentation level?

Reply:
I think, also i added some libraries (graphQL and swiftJSON)